wire-ios-sync-engine/Source/Calling/WireCallCenterV3.swift

```diff
@@ -950,6 +954,58 @@ extension WireCallCenterV3 {
         return conversationType
     }
 
+    /// Set MLSConferanceInfo to AVSWrapper in case this is a MLS conference
+    func setMLSConferanceInfoIfNeeded(for conversationId: AVSIdentifier, completion: @escaping (Bool) -> Void) {
```

Comment:
```suggestion
    /// Set MLSConferenceInfo to AVSWrapper in case this is a MLS conference
    func setMLSConferenceInfoIfNeeded(for conversationId: AVSIdentifier, completion: @escaping (Bool) -> Void) {
```